WalletWasabi.Tests/Helpers/CryptoHelpers.cs

```diff
@@ -0,0 +1,41 @@
+using NBitcoin.Secp256k1;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace WalletWasabi.Tests.Helpers
+{
+	public static class CryptoHelpers
+	{
+		public static readonly Scalar ScalarLargestOverflow = new Scalar(uint.MaxValue, uint.MaxValue, uint.MaxValue, uint.MaxValue, uint.MaxValue, uint.MaxValue, uint.MaxValue, uint.MaxValue);
+		public static readonly Scalar ScalarN = EC.N;
+		public static readonly Scalar ScalarEcnPlusOne = EC.N + Scalar.One;
+		public static readonly Scalar ScalarEcnMinusOne = EC.N + Scalar.One.Negate(); // Largest non-overflown scalar.
```

Comment:
this comment is really confusing since it only applies to the second summand in the expression

Reply:
oops, off by one. `Scalar.One.Negate() == EC.N + Scalar.One().Negate()`

Reply:
That's cool, it never clicked for me. I'll keep it like this for readability anyway 😄